File: parentControlApplication/src/main/java/ru/nekit/android/qls/parentControl/BootCompletedReceiver.java
package ru.nekit.android.qls.parentControl;

import android.annotation.SuppressLint;
import android.content.Context;
import android.content.Intent;
import android.support.v4.content.WakefulBroadcastReceiver;

public class BootCompletedReceiver extends WakefulBroadcastReceiver {

    public BootCompletedReceiver() {
    }

    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    public void onReceive(Context context, Intent intent) {
        startWakefulService(context, ParentControlService.getStartIntent(context));
    }
}

File: questPresentation/src/main/java/ru/nekit/android/qls/lockScreen/receiver/BootCompletedReceiver.kt
package ru.nekit.android.qls.lockScreen.receiver

import android.annotation.SuppressLint
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import ru.nekit.android.qls.lockScreen.LockScreen

class BootCompletedReceiver : BroadcastReceiver() {

    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    override fun onReceive(context: Context, intent: Intent) {
        LockScreen.getInstance().startOnBootComplete()
    }
}